src/nupic/ntypes/Value.cpp

```diff
@@ -29,17 +29,17 @@
 
 using namespace nupic;
 
-Value::Value(boost::shared_ptr<Scalar> &s) {
+Value::Value(std::shared_ptr<Scalar> &s) {
```

Comment:
could have just removed the prefix, and appropriate method would be used based on includes. But leave it as is.